src/static_handler.rs

```diff
@@ -5,6 +5,8 @@ use std::error::Error;
 use std::fmt;
 use time::{self, Timespec};
 
+use std::os::unix::prelude::MetadataExt;
```

Comment:
@alexcrichton any chance you want to toss that `MTime` type into its own crate so we can depend on it? It would be great to avoid duplication.

Thanks for the tips, especially on the windows behavior.